app/assets/stylesheets/grader.scss

```diff
@@ -649,6 +649,118 @@ ul.annotation_text_list {
     }
   }
 
+  .mark_checkbox {
+    font-weight: bold;
+    font-size: 11px;
+    float: right;
+
+    .mark_grade_input_checkbox {
+      font-size: 1em;
+      margin: 1px 1px 4px;
+      min-width: 0;
+      padding: 2px;
+    }
+  }
+
+  .old_mark {
+    font-size: 11px;
+    float: right;
+    padding-right: 10px;
+
+    &.old_mark_unreleased {
+      padding-top: 4px;
+    }
+  }
+
+  .mark_grade_input {
+    &.remarked {
+      background-color: #81d881;
+    }
+
+    &.not_remarked:not([disabled]) {
+      background-color: white;
+    }
+  }
+
+}
+
+/* ============= */
+
+/* Checkbox Criterion Styles */
+
+.checkbox_criterion {
+  margin-top: -4px;
+
+  .criterion_expand {
+    float: left;
+    cursor: pointer;
+    font-size: 1.2em;
+    width: 1em;
+    padding-top: 2px;
+    height: 20px;
+  }
+
+  &.expanded {
+    .criterion_info {
+      display: block;
+    }
+  }
+
+  &.not_expanded {
+    .criterion_info {
+      display: none;
+    }
+  }
+
+  .unexpandable {
+    .criterion_expand {
+      visibility: hidden;
+      cursor: auto !important;
+    }
+
+    .title_div_level {
+      cursor: auto !important;
+    }
+  }
+
+  .criterion_info {
+    padding-left: 10px;
+    padding-bottom: 5px;
+    margin-left: 5px;
+    float: left;
+    clear: both;
+  }
+
+  .title_div_level {
+    padding-top: 2px;
+    font-size: 12px;
+    float: left;
+    padding-right: 10px;
+    font-weight: bold;
+    padding-bottom: 6px;
+
+    &.released {
+      padding-left: 15px;
+    }
+
+    &.unreleased {
+      cursor: pointer;
+    }
+  }
+
+  .mark_checkbox {
+    font-weight: bold;
+    font-size: 11px;
+    float: right;
+
+    .mark_grade_input{
+      font-size: 1em;
+      margin: 1px 1px 4px;
+      min-width: 0;
+      padding: 2px;
+    }
+  }
+
```

Comment:
**Note to self:** The above may not be needed, I will check this as I'm doing the CSS handling at the end.